### PR TITLE
docs: addition of method documentation

### PR DESCRIPTION
**Pull request by JavierDelgadoFernandez**
* Improves method documentation. (addresses #59)

Signed-off-by: Javier Delgado <javier.delgado.fernandez@cern.ch>

### CODE REVIEW

File: invenio_collections/models.py [hunk 1]
```diff
@@ -38,10 +38,13 @@ def __repr__(self):
                 'dbquery: {0.dbquery}>'.format(self))
 
     id = db.Column(db.Integer, primary_key=True)
+    """Collection identifier."""
 
     name = db.Column(db.String(255), unique=True, index=True, nullable=False)
+    """Collection name."""
 
     dbquery = db.Column(db.Text, nullable=True)
+    """Query that returns all the records grouped in this collection."""
```
Comment (jirikuncar) [inline invenio_collections/models.py]: what about: ... all records that belongs to this collection. ?

File: docs/api.rst [hunk 2]
```diff
@@ -25,6 +25,40 @@
...
+-----
+.. automodule:: invenio_collections.query
+   :members:
+
+Receivers
+---------
+.. automodule:: invenio_collections.receivers
+   :members:
+
+Utils
```
Comment (jirikuncar) [inline docs/api.rst]: Utilities

**Comment by jirikuncar** [comment]
@JavierDelgadoFernandez is it finished? We can use only labels (waffle.io) to mark PR as in work or in review.

**Comment by JavierDelgadoFernandez** [comment]
@jirikuncar Am I missing any module to document?

**Pull request merged**